packages/web/src/client/components/Video/VideoPlayer.tsx

```diff
@@ -114,6 +148,44 @@ export const VideoPlayer = ({
         muted
         playsInline
       />
+      <VideoButton
+        style={style}
+        onFocus={handleButtonFocus}
+        onBlur={handleButtonBlur}
```

Comment:
I would consider adding a visibly hidden span in the button that we change the content of depending on state.